docs/pages/dsHost_TestSpecification.md

```diff
@@ -0,0 +1,107 @@
+# HOST Test Document
+
+## Version History
+
+| Date(DD/MM/YY) | Comment       | Version |
+| -------------- | ------------- | ------- |
+| 01/03/2024     | First Release | 1.0.0   |
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Introduction](#introduction)
+- [Module Description](#module-description)
+- [Testing Scope](#testing-scope)
+
+## Acronyms, Terms and Abbreviations
+
+- `SoC`    - System On a Chip
+- `EDID`   - Extended Display Identification
+- `API`    - Application programming interface
+- `CPU`    - Central processing unit
+- `dsHost` - Device Settings Host
+- `HDMI`   - High-Definition Multimedia Interface
+
+## References
+
+- `EDID` specifications - [https://en.wikipedia.org/wiki/Extended_Display_Identification_Data](https://en.wikipedia.org/wiki/Extended_Display_Identification_Data)
+- Python `EDID` decoder library is available here - [https://pypi.org/project/pyedid/](https://pypi.org/project/pyedid/)
+
+## Introduction
+
+This document provides an overview of the testing requirements for the `dsHost` module.
+It outlines the scope of testing, objectives, testing levels and approaches, specific test requirements, and expected deliverables.
+
+Interface of the test is available in this link - [https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsHost.h](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsHost.h)
+
+## Module Description
+
+High level overview:
+
+- `dsHost` offers a range of `API`s for retrieving information about the platform.
+- Data is retrieved from the `SoC` and `HDMI`. This data is passed to the caller.
+- In order to retrieve `HDMI` information, an external device must be connected.
+
+## Testing Scope
+
+|#|Testing Scope|Description|
+|-|------------------|----------------|
+|1|[Retrieve CPU Temperature](#retrieve-cpu-temperature)|Test if the module correctly retrieves the `CPU` temperature.|
+|2|[Obtain SOC ID](#obtain-soc-id)|Test if the module successfully obtains the 8-byte `SoC` ID programmed to the CHIP One Time Programmable area.|
+|3|[Fetch Host EDID](#fetch-host-edid)|Test if the module fetches the host `EDID` along with its length|
+-----------
+
+## Retrieve CPU Temperature
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Invoke the module to retrieve `CPU` temperature, verify the returned temperature falls within valid operating range and ensure the `CPU` is in a normal operating state.|dsGetCPUTemperature|Y|N|N|

```

Comment:
for L3 check if we can use heat/cold chambers and retrieve the temperature information

Reply:
Updated, please review the new test case.